source/reporters/ut_junit_reporter.tpb

```diff
@@ -46,16 +47,22 @@ create or replace type body ut_xunit_reporter is
       if a_test.result = ut_utils.gc_error then
         self.print_text('<error>');
         self.print_text('<![CDATA[');
-        self.print_clob(ut_utils.table_to_clob(a_test.get_error_stack_traces()));
+        self.print_clob(
+        replace(replace(ut_utils.table_to_clob(a_test.get_error_stack_traces()),l_cdata_tag_start,''),l_cddata_tag_end,'')
```

Comment:
why do we need to do this replace?

Reply:
In rare occasions when cddata string is part of error message. It then breaks xml format.